intermediate_source/tensorboard_profiler_tutorial.py

```diff
@@ -287,28 +289,46 @@ def train(data):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #
 # - Memory view
-# To profile memory, please add ``profile_memory=True`` in arguments of ``torch.profiler.profile``.
+# To profile memory, ``profile_memory`` must be set to ``True`` in arguments of ``torch.profiler.profile``.
 #
-# Note: Because of the current non-optimized implementation of PyTorch profiler,
-# enabling ``profile_memory=True`` will take about several minutes to finish.
-# To save time, you can try our existing examples first by running:
+# You can try it by using existing example on Azure
 #
 # ::
 #
-#     tensorboard --logdir=https://torchtbprofiler.blob.core.windows.net/torchtbprofiler/demo/memory_demo
+#     pip install azure-storage-blob
+#     tensorboard --logdir=https://torchtbprofiler.blob.core.windows.net/torchtbprofiler/demo/memory_demo_1_10
 #
-# The profiler records all memory allocation/release events during profiling.
-# For every specific operator, the plugin aggregates all these memory events inside its life span.
+# The profiler records all memory allocation/release events and allocator's internal state during profiling.
+# The memory view consist of three components as shown in the following.
 #
 # .. image:: ../../_static/img/profiler_memory_view.png
 #    :scale: 25 %
 #
+# The components are memory curve graph, memory events table and memory statistics table, from top to bottom, respectively.
+#
 # The memory type could be selected in "Device" selection box.
-# For example, "GPU0" means the following table only shows each operator’s memory usage on GPU 0, not including CPU or other GPUs.
+# For example, "GPU0" means the following table only shows each operator's memory usage on GPU 0, not including CPU or other GPUs.
+#
+# The memory curve shows the trends of memory consumption. You can left click and drag on it to select events in the
```

Comment:
Add explanation of "Allocated" and "Reserved"?

Reply:
done